src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -33,7 +33,7 @@ public class FindCommand extends Command {
             + "[" + PREFIX_STUDY_GROUP_TAG + "STUDY_GROUP_TAG ...]\n"
             + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "alice bob charlie";
 
-    public static final String MESSAGE_NO_CRITERIA = "At least one criteria to find by must be provided.";
+    public static final String MESSAGE_NO_CRITERIA = "Provide at least one criteria to find!";
```

Comment:
Made use of imperative mood here #CS2101 🔥

Reply:
Nice! Do all our messages also follow this ? 🤔

Reply:
So far yes, I also updated the one in `EditCommand` as well, just that it's under the `edit` PR (merged a few seconds ago)~